src/themes/DarkTheme.js

```diff
@@ -7,12 +7,19 @@ const colors = {
   brandLight: '#624F86',
   brandDark: '#542D9B',
   sunsetOrange: '#F94C4C',
+  lima: '#50BF17',
+  sushi: '#70C843',
   tuna: '#3A3A43',
   shark: '#2D2E33',
   mako: '#464651',
   blueHaze: '#CBCEDE',
   athensGray: '#E9EAED',
   waterloo: '#878891',
+  asparagus: '#6DA450',
+  cinnabar: '#E14848',
+  coralTree: '#A96868',
+  redOrange: '#F83333',
+  monza: '#D80707',
```

Comment:
All of those colors, except `lima` (I guess) aren't necessary. Please, leave either `lima` or `sushi` (one which is our main green) and use `lightenDarkenColor` helper to get the shade you need, instead of all other 6 new colors in the palette.